scripts/astral.zs

```diff
@@ -1,5 +1,13 @@
 import crafttweaker.oredict.IOreDict;
 
+//More interesting cake transmutation
+mods.astralsorcery.LightTransmutation.removeTransmutation(<minecraft:cake>, false);
+for meta in 0 to 4 {    
+    //thanks to AstralSorcery's stupid ct integration we must register it for any possible block state 
+    mods.astralsorcery.LightTransmutation.addTransmutation(<minecraft:pumpkin>.withDamage(meta), <harvestcraft:pumpkincheesecake>, 1000.0);
+    //fixme the additional recipes show up in jei with a missing texture 
```

Comment:
I tried removing the recipes that contain the missing textures from jei:

```suggestion
    val item = <minecraft:pumpkin>.withDamage(meta);
    mods.astralsorcery.LightTransmutation.addTransmutation(item, <harvestcraft:pumpkincheesecake>, 1000.0);
    if (meta > 0) {
        //try to hide the additional states from JEI as they display as missing texture
        mods.jei.JEI.removeAndHide(item);
```

But no changes. Even though `crafttweaker.log` states that the items were removed.